### ISSUE TITLE
Difference in reliability w/ SubtitleEdit+Tesseract? 

### ISSUE THREAD

**Issue by scr4tchy** [ratoaq2/pgsrip#14]
Hi!

Really excited to see this tool - fits amazingly as a tdarr plugin too!

I noticed while parsing my first English PGS (1h15min) that it detected 530 strings, whereas SubtitleEdit + Tesseract 5.3.0 detected 783 strings (and had great accuracy on them). I felt a bit surprised considering that both use Tesseract 5 and the performance should theoretically be really good regardless of the dataset given it's just working with bare English, black on white, straight. I noted that when subtitles are missed, the previous subtitles would stick around for a long time.

Do you have any ideas from your experience?

**Comment by viown** [comment]
I have this issue as well where some dialogue is missing from the ripped SRT file. I'm not sure if it's related but it seems to happen with segments that are in a different position from the center.

@ratoaq2 Here is a sample containing the original SUP file, the ripped pgsrip SRT file, and a Subtitle Edit version (which does not have any dialogue missing) that may help diagnose the issue. 

[subtitle.zip](https://github.com/ratoaq2/pgsrip/files/10873216/subtitle.zip)

**Comment by viown** [comment]
I tested out an older version of pgsrip (v0.1.4) and the issue does not happen there and all dialogue is extracted properly as expected. So it must be a recent change that causes this.

**Comment by scr4tchy** [comment]
Good observation.

On another media:
- pgsrip latest yields 530 strings (525 unique)
- pgsrip v0.1.4 yields 1131 strings (1104 unique)
- subtitle edit yields  1132 strings (1132 unique)

**Comment by ratoaq2** [comment]
Thanks for reporting it and providing information to reproduce it. I'm doing a release with the fix

**Comment by mrwunderbar666** [comment]
Hi,

thanks for making this great tool! 

I am running into similar issues where SubtitleEd
[example_subs.zip](https://github.com/ratoaq2/pgsrip/files/12657405/example_subs.zip)
it extracts more / better subtitles from PGS than pgsrip. I am not sure what causes the problem but SubtitleEdit found 463 strings, but pgsrip only 242.

I attached a sample sup file where I encountered the issue 

Help would be appreciated
